Samples/VirtualCamera/VirtualCameraTest/VirtualCameraTest.vcxproj

```diff
@@ -0,0 +1,162 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project DefaultTargets="Build" ToolsVersion="15.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="..\packages\Microsoft.Windows.CppWinRT.2.0.210304.5\build\native\Microsoft.Windows.CppWinRT.props" Condition="Exists('..\packages\Microsoft.Windows.CppWinRT.2.0.210304.5\build\native\Microsoft.Windows.CppWinRT.props')" />
+  <PropertyGroup Label="Globals">
+    <CppWinRTOptimized>true</CppWinRTOptimized>
+    <CppWinRTRootNamespaceAutoMerge>true</CppWinRTRootNamespaceAutoMerge>
+    <CppWinRTGenerateWindowsMetadata>true</CppWinRTGenerateWindowsMetadata>
+    <MinimalCoreWin>true</MinimalCoreWin>
+    <VCProjectVersion>15.0</VCProjectVersion>
+    <ProjectGuid>{347f4395-68f6-4936-af83-2206f42557bf}</ProjectGuid>
+    <Keyword>Win32Proj</Keyword>
+    <RootNamespace>VirtualCameraTest</RootNamespace>
+    <WindowsTargetPlatformVersion Condition=" '$(WindowsTargetPlatformVersion)' == '' ">10.0.19041.0</WindowsTargetPlatformVersion>
+    <WindowsTargetPlatformMinVersion>10.0.17134.0</WindowsTargetPlatformMinVersion>

```

Comment:
>10.0.17134.0 [](start = 37, length = 12)

nit: I think we should use latest LTS release SDK, RS5 = 17763

Reply:
same goes for other projects in this sln

---
In reply to: [614118346](https://github.com/microsoft/Windows-Camera/pull/17#discussion_r614118346) [](ancestors = 614118346)

Reply:
yes in process of installing the latest sdk and removing the props with sdk reference.

Reply:
21364 v.s. 21361? what is the diff and should it be exlusively 21364?